project-clone/ssh-askpass.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+PASSPHRASE_FILE_PATH="/etc/ssh/passphrase"
+if [ ! -f $PASSPHRASE_FILE_PATH ]; then
+    echo "Error: passphrase file is missing in the '/etc/ssh/' directory" 1>&2
+    exit 1
```

Comment:
@vinokurig could you elaborate a bit? why do we exit 1 if the optional passphrase is missing?

Reply:
Passphrase is optional because we do not need to specify passphrase if the ssh key is generated with an empty passphrase, but if the ssh key is generated **WITH** a passphrase we do not consider the passphrase as optional.